### PR TITLE
Add new event demultiplexor FBType E_DEMUX8 with 8 output events and transition logic.

### PR DESCRIPTION
**Pull request by franz-hoepfinger-4diac**

### CODE REVIEW

**Comment by azoitl** [comment]
Better indicate in the version history that you copy modified a block I did.

**Comment by diplfranzhoepfinger** [comment]
true, better, i will make a revision.

**Comment by diplfranzhoepfinger** [comment]
@azoitl revision done.

File: data/typelibrary/events-1.0.0/typelib/E_DEMUX8.fbt [hunk 1]
```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<FBType Name="E_DEMUX8" Comment="Event demultiplexor 8 Events">
+	<Identification Standard="61499-1 Annex A" Description="Copyright (c) 2017 fortiss GmbH&#10; &#10;This program and the accompanying materials are made&#10;available under the terms of the Eclipse Public License 2.0&#10;which is available at https://www.eclipse.org/legal/epl-2.0/&#10;&#10;SPDX-License-Identifier: EPL-2.0" >
+	</Identification>
+	<VersionInfo Organization="fortiss GmbH" Version="1.0" Author="Alois Zoitl" Date="2017-09-22" Remarks="initial API and implementation and/or initial documentation">
+	</VersionInfo>
+	<VersionInfo Organization="HR Agrartechnik GmbH" Version="1.1" Author="Franz Höpfinger" Date="2024-07-15" Remarks="copy and make a DEMUX8">
```
Comment (azoitl) [inline data/typelibrary/events-1.0.0/typelib/E_DEMUX8.fbt]: It would be nice to see where you copied it from

**Review by azoitl — CHANGES_REQUESTED** [review]

File: data/typelibrary/events-1.0.0/typelib/E_DEMUX8.fbt (hunk 1 above)
Comment (franz-hoepfinger-4diac) [reply data/typelibrary/events-1.0.0/typelib/E_DEMUX8.fbt]: yep, updated.

**Review by azoitl — APPROVED** [review]

**Pull request merged**